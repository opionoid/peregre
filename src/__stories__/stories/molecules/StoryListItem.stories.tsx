import { Story, Meta } from '@storybook/react'
import {
  StoryListItem,
  IStoryListItemProps,
} from 'src/components/stories/molecules/StoryListItem'
import MOCK_STORY_LIST_ITEM_PROPS from 'src/__mocks__/components/stories/molecules/StoryListItem'

export default {
  title: 'components/stories/molecules/StoryListItem',
  component: StoryListItem,
} as Meta

const Template: Story<IStoryListItemProps> = (args) => (
  <StoryListItem {...args} />
)
export const Default = Template.bind({})
Default.args = {
  ...MOCK_STORY_LIST_ITEM_PROPS,
}
