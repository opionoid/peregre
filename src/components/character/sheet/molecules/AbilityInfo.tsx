import { IAbility } from "src/interfaces";

export const AbilityInfo: React.VFC<IAbility> = (props) => (
  <>
    <h3>{props.name}</h3>
    <p>{props.description}</p>
  </>
)