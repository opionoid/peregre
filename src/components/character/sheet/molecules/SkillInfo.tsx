import { Icons } from "src/assets/icons";
import { ISkill } from "src/interfaces";

export const SkillInfo: React.VFC<ISkill> = (props) => (
  <>
    <div style={{ display: 'flex', columnGap: '32px', alignItems: 'center' }}>
      <h3 style={{ minWidth: '25%' }}>{props.name}</h3>
      <p style={{ fontSize: '1.4rem', lineHeight: '1', margin: '0' }}>
        <img
          src={Icons.Depth}
          alt=""
          width={24}
          height={24}
          style={{ marginRight: '0.625rem' }}
        />
        {props.depth}
      </p>
    </div>
    <p>{props.description}</p>
  </>
)