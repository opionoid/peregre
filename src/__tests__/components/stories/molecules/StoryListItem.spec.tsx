import { StoryListItem } from 'src/components/stories/molecules/StoryListItem'
import MOCK_STORY_LIST_ITEM_PROPS from 'src/__mocks__/components/stories/molecules/StoryListItem'
import { render } from '@testing-library/react'

describe('stories/molecules', () => {
  const snapshot = render(<StoryListItem {...MOCK_STORY_LIST_ITEM_PROPS} />)

  it('default test', () => {
    expect(snapshot.baseElement).toMatchSnapshot()
  })
})
