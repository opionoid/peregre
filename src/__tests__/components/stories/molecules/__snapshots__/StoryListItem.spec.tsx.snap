// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stories/molecules default test 1`] = `
<body>
  <div>
    <section
      class="sc-bdvvaa"
    />
  </div>
</body>
`;
